pacemaker.spec.in

```diff
@@ -402,6 +403,10 @@ rm -rf %{buildroot}
 
 %post
 /sbin/chkconfig --add pacemaker || :
+%if %{cman_allow} && %{with cman}
+# see https://bugzilla.redhat.com/1116401
+/usr/sbin/ccs_update_schema >/dev/null 2>&1 || :
+%endif
```

Comment:
It seems to me that install time is not the right place to do this; there's nothing wrong with installing the software with an invalid cluster.conf, it's starting it that matters. And from the discussion, it seems that this is a vanishingly small issue.